### PR TITLE
Wording - LightningNetwork, README, UseCase, Wallet

### PR DESCRIPTION
**Pull request by doynovbps**
Changed wording, grammar to improve clarity, consistency across LightningNetwork, README, UseCase, Wallet.MD.

### CODE REVIEW

File: LightningNetwork.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ After deploying BTCPay Server, you may want to experiment with an innovative sec
 
 This guide will show you how to set up your Lightning Network node in BTCPay and guide you through the basics.
 
-<strong>Before you proceed, please understand that Lightning Network is still in the experimental stage. Do not put the money you can't afford to lose. There is a high risk of you losing the money. </strong>
+<strong>Before you proceed, please understand that Lightning Network is still in the experimental stage. Do not use Lightning Network for money you cannot afford to lose. There is a high risk of you losing money with the Lightning Network. </strong>
```
Comment (pavlenex) [inline LightningNetwork.md:7]: > Do not use Lightning Network for money you cannot afford to lose

This sounds weird to me. @britttttk what do you think?

File: UseCase.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ The software usage is by no means limited to the groups of users mentioned in th
 
 By choosing BTCPay Server to process payments, merchants are:
 
-* Saving money (no fees, no subscriptions)
-* Cutting out the middle-man (Payments go directly to their wallet)
+* Saving money (no fees, no subscriptions; a typical fee for a fiat payment processor in the USA is 2.9% + $0.30/transaction!)
```
Comment (pavlenex) [inline UseCase.md:13]: Redundant information, please remove this and leave as it was

File: UseCase.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ Charities, non-profits, content creators, and other organizations that want to a
 
 Benefits of using BTCPay for accepting donations:
 
-* Saving money (no fees, no subscriptions)
+* Saving money (no fees, no subscriptions; crowfunding campaigns have 5% fees, in addition to payment processor fees of 3%!)
```
Comment (pavlenex) [inline UseCase.md:53]: Redundant please remove this

**Comment by doynovbps** [comment]
Agreed, on lightningnetwork.md sounds like my proposed change didn't help. Wording was off before my changes as well.

On fees I added the US processing fees as an illustration. The mainstream narrative is that BTC fees are too high. I wanted to highlight that payment processor fees are high and not considered / an afterthought in any e purchase (i.e. the cost of doing business). Still want this removed from UseCase?

Maybe this is a new document BTC network fees vs. traditional payment processor fees as opposed to described in UseCase.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by pavlenex** [comment]
I think putting such data shouldn't be placed in docs. It changes frequently and really depends on location. Hence I am against putting it here.

File: LightningNetwork.md (hunk 1 above)
Comment (Zaxounette) [reply LightningNetwork.md:7]:
```suggestion
<strong>Before you proceed, please understand that Lightning Network is still in the experimental stage. There are high risks. Do not use Lightning Network with funds you can't afford to lose.</strong>
```

**Comment by pavlenex** [comment]
Closing due to inactivity, if you're willing to work on this in the future, feel free to re-open.

**Pull request closed without merging**